tomcat-server/src/main/scala/org/http4s/tomcat/server/TomcatBuilder.scala

```diff
@@ -159,15 +163,17 @@ sealed class TomcatBuilder[F[_]] private (
   def mountService(service: HttpRoutes[F], prefix: String): Self =
     mountHttpApp(service.orNotFound, prefix)
 
-  def mountHttpApp(service: HttpApp[F], prefix: String): Self =
+  def mountHttpApp(httpApp: HttpApp[F], prefix: String): Self =
     copy(mounts = mounts :+ Mount[F] { (ctx, index, builder, dispatcher) =>
-      val servlet = new AsyncHttp4sServlet(
-        service = service,
-        asyncTimeout = builder.asyncTimeout,
-        servletIo = builder.servletIo,
-        serviceErrorHandler = builder.serviceErrorHandler,
-        dispatcher = dispatcher,
-      )
+      val chunkSize = builder.servletIo match {
+        case BlockingServletIo(chunkSize) => chunkSize
+        case NonBlockingServletIo(chunkSize) => chunkSize
+      }
```

Comment:
If we change the builder property from `servletIo: ServletIo` to `chunkSize: Long`, this melts away.